docs/admin/extensible-admission-controllers.md

```diff
@@ -126,6 +126,11 @@ initializers:
           - pods
 ```
 
+Once this `initializerConfiguration` named `example-config` is created, the
```

Comment:
@caesarxuchao there were about 4 other occurrences of `metadata.initializers` that doesn't mention `.pending`. I changed them all. PTAL again.